src/test/java/core/basesyntax/strategy/handler/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static FruitTransaction fruitTransaction;
+    private static BalanceOperationHandler balanceOperationHandler;
```

Comment:
```suggestion
    private static OperationHandler balanceOperationHandler;
```
the same for other implentations test classes